packages/sanity/src/core/store/_legacy/document/document-pair/editState.ts

```diff
@@ -21,10 +21,24 @@ export interface EditStateFor {
   type: string
   transactionSyncLock: TransactionSyncLockState | null
   draft: SanityDocument | null
-  // TODO: Rename -> `public`
   published: SanityDocument | null
+  version: SanityDocument | null
+  /**
+   * Whether live edit is enabled. This may be true for various reasons:
+   *
+   * - The schema type has live edit enabled.
+   * - A version of the document is checked out.
+   */
   liveEdit: boolean
+  /**
+   * Whether the schema type has live edit enabled.
+   */
+  liveEditSchemaType: boolean
```

Comment:
I wonder if some users have implemented this in a similar way to our liveEditBadge.
Maybe liveEdit should return the same as before and it could be better to have a `versionEdit` ? And a note saying versionEdit emulates a liveEdit, because version documents don't follow the drafts. convention

Reply:
I like this approach, if it's a big change it can also be a follow up